extensions/arc/package.json

```diff
@@ -261,6 +261,20 @@
                           "label": "%arc.data.controller.details.description%",
                           "labelWidth": "600px"
                         },
+                        {
+                          "type": "options",
+                          "label": "%arc.data.controller.connectivity.mode%",
+                          "required": true,
+                          "variableName": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_CONNECTIVITY_MODE",
+                          "options": {
+                            "values": [
+                              "%arc.data.controller.indirect%",
```

Comment:
These are currently going to break if the user has a non-English language since you're passing that directly to the CLI. Probably just easiest to use English names for them directly, although you could also set separate displayNames for these as well

Reply:
Added a screenshot of the UI! Sorry for the delay on response to this comment, I have been testing this out with AKS clusters. 

By "since you're passing that directly to the CLI", are you saying that this part below is the issue?:
run_command(f'az arcdata dc create --connectivity-mode {**arc_data_controller_connectivity_mode**}  ...."
Could you help clarify what you meant by "use English names for **them**"?

Reply:
Yes. Because you're localizing these strings if someone was to install a non-EN langpack then we'd pass the localized strings to the CLI (and I'm assuming here that the CLI wouldn't be able to understand those)

Reply:
So are you suggesting that I could just put them in directly like this:
`{
                          "type": "options",
                          "label": "%arc.data.controller.connectivity.mode%",
                          "required": true,
                          "variableName": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_CONNECTIVITY_MODE",
                          "options": {
                            "values": [
                              "Indirect",
                              "Direct"
                              ],
                              "defaultValue": "Indirect",
                              "optionsType": "radio"
                          }
},`

Reply:
That works but you could also consider using the CategoryValues to give them both a display name (localized) and a name (underlying value passed to CLI) instead if you wanted to keep the UI localized. I just added a sample here for you to reference : https://github.com/microsoft/azuredatastudio/pull/17734